sample-app/build.gradle

```diff
@@ -70,6 +70,11 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.6.2'
```

Comment:
why LeakCanary is introduced on this PR?

Reply:
by mistake.
Should i remove it and add it it to a different pr?

Reply:
yep